service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantMidEventControllerTest.java

```diff
@@ -64,7 +64,7 @@ void shouldReturnNoErrorsWhenThereIsNewApplicantAndPbaNumberIsNull() {
                         .email(EmailAddress.builder().build())
                         .build())
                     .build()),
-                "solicitor", Solicitor.builder().build()))
+                "solicitor", Solicitor.builder().email("email@example.com").build()))
```

Comment:
Could we update these tests so that instead of using a map we are using a case data object. The post methods accept case data and it has better type safety over a map which accepts any object.